system/Database/SQLite3/Table.php

```diff
@@ -200,25 +201,78 @@ public function modifyColumn(array $field)
      *
      * @return Table
      */
-    public function dropForeignKey(string $column)
+    public function dropForeignKey(string $foreignName)
     {
         if (empty($this->foreignKeys)) {
             return $this;
         }
 
-        for ($i = 0; $i < count($this->foreignKeys); $i++) {
-            if ($this->foreignKeys[$i]->table_name !== $this->tableName) {
-                continue;
-            }
+        if (isset($this->foreignKeys[$foreignName])) {
+            unset($this->foreignKeys[$foreignName]);
+        }
 
-            // The column name should be the first thing in the constraint name
-            if (strpos($this->foreignKeys[$i]->constraint_name, $column) !== 0) {
-                continue;
-            }
+        return $this;
+    }
+
+    /**
+     * Add a foreign key
+     *
+     * @return Table
```

Comment:
Added this method so foreign keys can be created after table creation and used with new processIndexes() method.